client/components/forms/us-state-selector.jsx

```diff
@@ -1,102 +1,90 @@
 /**
```

Comment:
cc @obenland re: #8202 . I wasn't aware this component existed. A bit strange that we have a hard-coded state selector, though I guess it saves a network request. Might be a good candidate for replacement by [proposed `<StateSelect />` block](https://github.com/Automattic/wp-calypso/pull/8202#issuecomment-249667469).